contributing/development/configuring_an_ide/kdevelop.rst

```diff
@@ -78,7 +78,7 @@ Debugging the project
 - Select **Executable** option and specify the path to your executable located in
   the ``<Godot root directory>/bin`` folder. The name depends on your build configuration,
   e.g. ``godot.linuxbsd.editor.dev.x86_64`` for 64-bit LinuxBSD platform with
-  ``platform=editor`` and ``dev_build=yes``.
+  ``platform=linuxbsd``, ``target=editor`` and ``dev_build=yes``.
```

Comment:
```suggestion
  ``platform=linuxbsd``, ``target=editor``, and ``dev_build=yes``.
```
Please use [Oxford comma](https://docs.godotengine.org/en/latest/contributing/documentation/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything).